templates/arbiter_outputs_entry.py

```diff
@@ -0,0 +1,247 @@
+from __future__ import annotations
+
+from pydm import Display
+from pydm.widgets.channel import PyDMChannel
+from qtpy.QtCore import QObject, QTimer, Signal
+
+
+class ArbiterRow(Display):
```

Comment:
Brief docstrings for all these new classes + methods would be nice, though since they are reasonably straightforwardly-named we could certainly get by without them.

Reply:
This is worth doing, I'll pan through my own diff and fill the docs in.

Reply:
Added in https://github.com/pcdshub/pmps-ui/pull/96/commits/155ae90e2326bebf59bbc412d475a337d39d90cf